tests/devapps/Managed Identity apps/ManagedIdentityAppVM/Program.cs

```diff
@@ -19,24 +24,62 @@
 
     try
     {
-        var result = await mi.AcquireTokenForManagedIdentity(scope)
-            .ExecuteAsync().ConfigureAwait(false);
+        for (int i = 1; i <= 3; i++)
```

Comment:
not a fan of this, makes the code hard to read. Consider duplicating the `AcquireToken` code.